primitives/runtime-interface/tests/ui/no_generic_parameters.stderr

```diff
@@ -1,9 +1,3 @@
-error: Generic parameters not supported.
- --> $DIR/no_generic_parameters.rs:5:10
-  |
-5 |     fn test<R>() {}
```

Comment:
Why did we lost this?

Reply:
Because now we first check if trait has any generic and returning immediately if it has, I'll add another failing test for methods